### PR TITLE
Enabling tech preview features in the console

### PR DESCRIPTION
**Pull request by ahardin-rh**
Continuing work from https://github.com/openshift/openshift-docs/pull/2641

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape PTAL

File: install_config/web_console_customization.adoc [hunk 1]
```diff
@@ -188,7 +188,7 @@ The available extension points are:
 * `*nav-user-dropdown*` - the user dropdown menu, visible at desktop screen widths
 * `*nav-dropdown-mobile*` - the single menu for top navigation items at mobile screen widths
 
-The following example extends the `*nav-help-dropdown*` menu, with a name of '<myExtensionModule>': 
+The following example extends the `*nav-help-dropdown*` menu, with a name of '<myExtensionModule>':
```
Comment (adellape) [inline install_config/web_console_customization.adoc]: Backticks around <myExtensionModule>

File: install_config/web_console_customization.adoc [hunk 2]
```diff
@@ -334,6 +334,38 @@ endif::[]
 
 endif::[]
 
+[[enable-tech-preview-feature]]
+=== Enabling Features in Tech Preview
```
Comment (adellape) [inline install_config/web_console_customization.adoc]: s/Tech/Technology/

File: install_config/web_console_customization.adoc [hunk 3]
```diff
@@ -334,6 +334,38 @@ endif::[]
 
 endif::[]
 
+[[enable-tech-preview-feature]]
```
Comment (adellape) [inline install_config/web_console_customization.adoc]: Add "web-console" somewhere in the ID?

File: install_config/web_console_customization.adoc [hunk 4]
```diff
@@ -334,6 +334,38 @@ endif::[]
...
+=== Enabling Features in Tech Preview
+
+Sometimes features are available in tech preview. By default, these features are
```
Comment (adellape) [inline install_config/web_console_customization.adoc]: s/tech preview/Technology Preview/

Also link the words to https://access.redhat.com/support/offerings/techpreview.

File: install_config/web_console_customization.adoc [hunk 5]
```diff
@@ -334,6 +334,38 @@ endif::[]
...
+hidden from end users.
```
Comment (adellape) [inline install_config/web_console_customization.adoc]: Maybe:

s/hidden/disabled in the web console and hidden/

File: install_config/web_console_customization.adoc [hunk 6]
```diff
@@ -334,6 +334,38 @@ endif::[]
...
+
+Features currently in tech preview include:
```
Comment (adellape) [inline install_config/web_console_customization.adoc]: s/tech preview/Technology Preview/

Comment (adellape) [inline install_config/web_console_customization.adoc]: Also maybe:

s/Features/Web console features/

File: install_config/web_console_customization.adoc [hunk 7]
```diff
@@ -334,6 +334,38 @@ endif::[]
...
+
+* Pipelines - Enabling this will add the *Pipelines* navigation item underneath
+ the *Builds* menu.
+
+To enable a feature:
```
Comment (adellape) [inline install_config/web_console_customization.adoc]: a Technology Preview feature

File: install_config/web_console_customization.adoc [hunk 8]
```diff
@@ -334,6 +334,38 @@ endif::[]
...
+[[enable-tech-preview-feature]]
+=== Enabling Features in Tech Preview
+
+Sometimes features are available in tech preview. By default, these features are
+hidden from end users.
+
+Features currently in tech preview include:
+
+* Pipelines - Enabling this will add the *Pipelines* navigation item underneath
+ the *Builds* menu.
```
Comment (adellape) [inline install_config/web_console_customization.adoc]: We need to show probably here that `pipelines` is what they need to put in for <feature_name> below for this feature. Maybe a table? Feature name, description, script value?

**Review by adellape — CHANGES_REQUESTED** [review]

**Comment by ahardin-rh** [comment]
@adellape Thanks! All fixed.

File: install_config/web_console_customization.adoc (hunk 1 above)
Comment (adellape) [inline install_config/web_console_customization.adoc]: Missed this one.

**Review by adellape — APPROVED** [review]

File: install_config/web_console_customization.adoc (hunk 1 above)
Comment (ahardin-rh) [inline install_config/web_console_customization.adoc]: whoops! fixed!

**Pull request merged**